codecademy/introduction_to_classes.py

```diff
@@ -10,15 +10,16 @@ def __init__(self, name, color, flavor, poisonous, weight):
     def description(self):
         return "I'm a %s %s and I taste %s." % (self.color, self.name, self.flavor)
 
-    def is_edible(self, type_posinous):
-        assert type(type_posinous) == bool, "Error type, please enter boolean"
+    def is_edible(self, type_poisinous):
+        assert type(type_poisinous) == bool, "Error type, please enter boolean"
         if not self.poisonous:
             return True
         else:
             return False
 
     def bite_it(self, bite_size):
-        assert type(bite_size) in (int, float) and bite_size > 0, "{} error type, please enter integer".format(bite_size)
+        assert type(bite_size) in (int, float) and bite_size > 0,\
+            "{} error type, please enter integer".format(bite_size)
```

Comment:
no need to make short lines.
We will be OK with long lines if that doesn't really affect readability